README.md

```diff
@@ -0,0 +1,96 @@
+# Net Agent
+
+[![Crates.io](https://img.shields.io/crates/v/net-agent.svg)](https://crates.io/crates/net-agent)
+[![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+`net-agent` is a binary crate that allows you to capture network traffic and load it to [my.netstalker.io](https://my.netstalker.io) for further analysis.
```

Comment:
Net-Agent (Net-agent) is a standalone bin crate (app?).
It is not connected to [my.netstalker.io](https://my.netstalker.io) anyhow.